toast-tk-web-agent/src/main/java/io/toast/tk/agent/guice/UiModule.java

```diff
@@ -0,0 +1,19 @@
+package io.toast.tk.agent.guice;
+
+import com.google.inject.AbstractModule;
+import io.toast.tk.agent.ui.ConfigPanel;
+import io.toast.tk.agent.ui.PropertiesHolder;
+import io.toast.tk.agent.ui.provider.ConfigPanelProvider;
+import io.toast.tk.agent.ui.provider.PropertiesProvider;
+
+import java.util.Properties;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Properties'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)